### ISSUE TITLE
Accessibility : WordPress Media Picker - Video Selection Issue

### ISSUE THREAD

**Issue by jd-alexander** [wordpress-mobile/WordPress-Android#11077]
<img src="https://user-images.githubusercontent.com/1509205/72281506-b6a31200-3608-11ea-811d-9e722ade0839.png" width="320">

### Expected behavior
"Video Selected" should be shown when a video selection occurs.

### Actual behavior
"Image Selected" is shown. 

### Steps to reproduce the behavior
1. Create post while TalkBack is enabled.
2. Add video from WordPress Media Picker. 
3. Select the video and listen to the announcement. 

Tested on Google Pixel 4 XL, Android 10, WPAndroid `develop` ()

**Comment by oguzkocer** [comment]
This issue is targeting the current frozen milestone which is about to be released. I am bumping it to `14.5` which is when a PR that lands to `develop` in the next 2 weeks will be released. If it's more urgent than that, please make sure to target `release/14.4` branch which will be available later today and update the milestone for the issue accordingly.
